cylc/flow/task_pool.py

```diff
@@ -85,6 +86,15 @@
 Pool = Dict['PointBase', Dict[str, TaskProxy]]
 
 
+# CLI prerequisite pattern: point/name:label
+REC_CLI_PREREQ = re.compile(
+    rf"({TaskID.POINT_RE})" +
+    rf"{TaskID.DELIM2}" +
+    rf"({TaskID.NAME_RE})" +
+    r':' + r'(\w+)'  # TODO: formally define qualifier RE?
+)
+
+
```

Comment:
`Tokens` parses qualifiers too (it calls them "selectors").

`Tokens(id_str, is_relative=True)`

Reply:
Done.